src/game/level.c

```diff
@@ -905,7 +905,7 @@ static void Level_CompleteSetup(int32_t level_num)
     Stats_ObserveRoomsLoad();
 
     // Must be called after all g_Anims, g_Meshes etc initialised.
-    Setup_AllObjects();
+    Object_SetupAllObjects();
```

Comment:
Is the second "object" redundant? I guess "Object_SetupAll" might be a little unclear.

Reply:
I don't have a strong opinion on either, maybe @lahm86 can chime in?

Reply:
Either is fine IMO. `Object_SetupAllObjects` is very clear as to what it does.